CHANGELOG.md

```diff
@@ -1,42 +1,59 @@
+# 1.4.0
+
+- Adds a new `-t` argument, to output only the types defined in an AVDL file, excluding interface definitions.
+
 # 1.3.30
+
 - @layout("array") feature for laying out structs as arrays
 
 # 1.3.29 (2018-12-07)
+
```

Comment:
Prettier decided to reformat the entire file 🙄 Happy to change it back if that's preferred.